purchase_orderline_images/reports/purchase_order_report.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchase_order_product_image"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath
+            expr="//table[hasclass('o_main_table')]/thead/tr/th[@name='th_taxes']"
+            position="before"
+        >
+            <t t-if="o.check_show_product_image_in_purchase_report()">
```

Comment:
afaiu it fails if there is no pic and that's why we need this PR with check on template?

Reply:
Hi, at first I didn't understand what it was about)
This module is in order to display the image in the report, and accordingly, so that it can be controlled, this check is added on the template itself